src/main/java/uk/gov/register/presentation/view/ItemView.java

```diff
@@ -26,4 +28,13 @@
     public Map<String, FieldValue> getContent() {
         return item.getFieldsStream().collect(Collectors.toMap(Map.Entry::getKey, itemConverter::convert));
     }
+
+    @Override
+    public CsvSchema csvSchema() {
+        CsvSchema.Builder schemaBuilder = new CsvSchema.Builder() ;
+        for (String value : getContent().keySet()) {
```

Comment:
Hmm, thinking about this I'm not sure it's quite right.

It might work for the `/item` resource at the moment, but for the `/records` (or later `/items`) resource, we need a single schema that will work for arbitrary records (or items).  This means we need to include all possible fields in the schema, even fields which aren't present in a particular item.

So, for example, if a `country` doesn't have an `end-date`, I think we should still have an `end-date` column in the CsvSchema.